go/run_lemon-mint3.sh

```
@@ -1,4 +1,4 @@
-cd go && \
-  go build -gccgoflags="-O3 -march=native -mtune=native -fgo-compiling-runtime -flto -finline-functions -fdevirtualize -fno-check-pointer-bounds -fno-go-check-divide-zero" -compiler gccgo -o ./../main ./cmd/lemon-mint3/main.go && \
-  cd .. && \
-  ./main
+cd go
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)